install/helm/open-match/templates/synchronizer.yaml

```diff
@@ -28,7 +28,11 @@ spec:
     app: {{ template "openmatch.name" . }}
     component: synchronizer
     release: {{ .Release.Name }}
-  type: {{ coalesce .Values.global.kubernetes.service.portType .Values.synchronizer.portType }}
+{{- $portType := coalesce .Values.global.kubernetes.service.portType .Values.synchronizer.portType -}}
```

Comment:
Synchronizer does not scale up? Only single pod - do we still need this.

Reply:
Good point, fixed.